### DISCUSSION TITLE
Does there support override style?

### DISCUSSION THREAD

**Discussion by jasper241024** [vuejs/vitepress#1024] [Q&A] [answered]
My question is that, my table component affected by `.vp-doc table`. So I wanna override this.

**Comment by brc-dd** [comment, marked as answer]
https://vitepress.vuejs.org/guide/theme-introduction.html#customizing-css, https://github.com/vuejs/vitepress/issues/199#issuecomment-1168325262

**Reply by warmthsea** [reply]
> [https://vitepress.vuejs.org/guide/theme-introduction.html#customizing-css，#199](https://vitepress.vuejs.org/guide/theme-introduction.html#customizing-css)（[评论）](https://github.com/vuejs/vitepress/issues/199#issuecomment-1168325262)

What about this?
![image](https://user-images.githubusercontent.com/45450994/180131236-82939c18-6ade-487a-8eda-533a331896be.png)

**Reply by brc-dd** [reply]
@warmthsea You'll have to increase its [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).

**Reply by warmthsea** [reply]
> @warmthsea你必须增加它的[特异性](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)。

sure, but it seems except `!important` no other way.

**Reply by brc-dd** [reply]
@warmthsea Try `.image.image` or `.image.image.image` -- it's a popular hack 😅

Here are other ones too: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#tips_for_handling_specificity_headaches

Something like `#app .image` or `#app .Layout .image` should also work.

**Reply by warmthsea** [reply]
> @warmthsea Try `.image.image` or `.image.image.image` -- it's a popular hack 😅
> 
> Here are other ones too: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#tips_for_handling_specificity_headaches
> 
> Something like `#app .image` or `#app .Layout .image` should also work.

ok，thanks

**Reply by jasper241024** [reply]
> https://vitepress.vuejs.org/guide/theme-introduction.html#customizing-css, [#199 (comment)](https://github.com/vuejs/vitepress/issues/199#issuecomment-1168325262)

Thanks. But my problem is that `.vp-doc` affects my custom components lib. It cause my table component style error.

**Reply by brc-dd** [reply]
@linxiaodi Read that comment I linked. You can add a `.postcssrc.js` file like the one I wrote there. Then you could do stuff like this:

```md

Here they will have styles.

<div class="vp-raw">
    Things inside vp-raw won't have styles.
</div>
```

**Reply by jasper241024** [reply]
Thanks for your help. It works. @brc-dd
